src/curateDict.ts

```diff
@@ -44,5 +45,71 @@ export default function curateDict(
   mappedDicomData.dict =
     dcmjs.data.DicomMetaDictionary.denaturalizeDataset(naturalData)
 
+  // Restore quarantined private tags directly to the final DICOM dict
```

Comment:
Could this be factored out into something like the get/set methods in lodash but taking into account the dicom requirements?  Seems like it could be useful in other places too.